qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -9,12 +9,13 @@
     vm_thread_contexts = "tc1"
     smp_fixed = 8
     vcpu_maxcpus = ${smp_fixed}
-    vm_thread_context_cpu-affinity = "1-7"
+    sandbox_error_message = "Setting CPU affinity failed: Operation not permitted"
```

Comment:
It is still for `qemu_sandbox = on`,  Is there any reason to set it for all ?

Reply:
Updated, thanks !